examples/BART/BART_introduction.ipynb

```diff
@@ -24,7 +24,7 @@
      "name": "stdout",
```

Comment:
<p>Bit of a side comment, also related to the discussion on discourse. The function signature has <code>X=None</code>,making it an optional parameter, but it doesn't look like it's optional  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/323/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Right, I will fix this.